### DISCUSSION TITLE
po-page-dynamic-detail

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1028] [Angular] [answered]
No Array<PoPageDynamicDetailField> não tem o atributo "mask"?

**Comment by jhonyeduardo** [comment, marked as answer]
Boa tarde @wsteixeira 

Não, vamos abrir uma issue para ajustar a documentação, mas atualmente não é possível informar mask.

Veja a documentação do [DynamicView](https://po-ui.io/documentation/po-dynamic-view)
